chainer/training/trainer.py

```diff
@@ -192,7 +192,9 @@ def extend(self, extension, name=None, trigger=None, priority=None,
         Args:
             extension: Extension to register.
             name (str): Name of the extension. If it is omitted, the
-                ``default_name`` attribute of the extension is used instead.
+                :attr:`Extension.name` attribute of the extension is used or
+                the :attr:`Extension.default_name` attribute of the extension
+                if ''name'' is also undefined.
```

Comment:
* `''name''` -> ``` `name` ```
* `also undefined` -> `is set to None or is undefined`

Reply:
Done.